reco/LST1_Hillas.py

```diff
@@ -210,26 +210,22 @@ def guess_type(filename):
             l = np.rad2deg(np.arctan2(hillas.length, foclen))
 
             #Calculate Timing parameters
-#            peak_time = units.Quantity(peakpos[chan])*units.Unit("ns")
-#            timepars = time.timing_parameters(geom.pix_x,geom.pix_y,clean,peak_time,hillas.psi)
-            pedcorrectedsamples[~window] = 0.0            
-            pedcorrectedsamples[pedcorrectedsamples < 0] = 0.001 # to avoid sum=0
-            ind = np.indices(data.shape)[2]
-            wmtime = np.average(ind, axis=2, weights=pedcorrectedsamples) # weighted mean time
- 
-            wmtime_ns = units.Quantity(wmtime[0])*units.Unit("ns")
-            timegrad, intcpt = time.timing_parameters(geom.pix_x,geom.pix_y,clean,wmtime_ns,hillas.psi)
-
-            # Calculate Time Gradient from shower head            
-            if np.sign(hillas.skewness) != 0 :
-                timegrad *= (-1.0) * np.sign(hillas.skewness)
-            print('time gradient : ',timegrad)
+            peak_time = units.Quantity(peakpos[chan])*units.Unit("ns")
+            timepars = time.timing_parameters(geom,clean,peak_time,hillas)
```

Comment:
are you in the end not including the time calculation at all? I think that the weighed average, if not much more time consuming, is a better way of calculating the arrival time of the photons rather than the peak time, but it would be nice if you could include this function either on ctapipe or in another module from this repo.

Reply:
I'm planning to move weighted mean calc. to somewhere in ctapipe.
I cannot see any modules to calculate arrival time except "peakpos",
so I may have to create it (like "time_extractor.py" under "image" dir?) & do PR...